### PR TITLE
Tweak Lambertian reflection description

### PR DESCRIPTION
**Pull request by hollasch**
My main object was to address some confusing wording of the last sentence of the first paragraph, but I also massed other parts of this paragraph.

### CODE REVIEW

File: books/RayTracingInOneWeekend.html [hunk 1]
```diff
@@ -2452,13 +2452,13 @@
...
-objects is the _Lambertian_ distribution, which has a scattering distribution that is propertional
-to $\cos (\phi)$, where $\phi$ is the angle from the normal. This means that a ray is most likely to
-scatter toward the normal and decreases in likelihood as the direction becomes more tangential. This
-distribution is notably no longer uniform, but the _Lambertian_ distribution does a better job of
-modeling objects in the real world than our previous uniform scattering. We can create this
-distribution by picking random points on the surface of the unit sphere, and then offsetting them
-along the surface normal.
+objects is the _Lambertian_ distribution, which scatters reflected rays in a manner that is
+propertional to $\cos (\phi)$, where $\phi$ is the angle between the reflected ray and the surface
+normal. This means that a ray is most likely to scatter in directions close to the normal, and less
```
Comment (trevordblack) [inline books/RayTracingInOneWeekend.html:2457]: Ugh. Now I want a graph to show this.

There's already an issue for book 3 that I was planning on addressing there, but maybe it should go here.

Regardless, it's _not_ going in alpha

File: books/RayTracingInOneWeekend.html [hunk 2]
```diff
@@ -2452,13 +2452,13 @@
...
 diffuse model, but we can definitely do better. A more accurate representation of real diffuse
-objects is the _Lambertian_ distribution, which has a scattering distribution that is propertional
-to $\cos (\phi)$, where $\phi$ is the angle from the normal. This means that a ray is most likely to
-scatter toward the normal and decreases in likelihood as the direction becomes more tangential. This
-distribution is notably no longer uniform, but the _Lambertian_ distribution does a better job of
-modeling objects in the real world than our previous uniform scattering. We can create this
-distribution by picking random points on the surface of the unit sphere, and then offsetting them
-along the surface normal.
+objects is the _Lambertian_ distribution, which scatters reflected rays in a manner that is
+propertional to $\cos (\phi)$, where $\phi$ is the angle between the reflected ray and the surface
```
Comment (trevordblack) [inline books/RayTracingInOneWeekend.html:2456]: lol, proportional [sic]

File: books/RayTracingInOneWeekend.html [hunk 3]
```diff
@@ -2452,13 +2452,13 @@
...
+normal. This means that a ray is most likely to scatter in directions close to the normal, and less
+likely to scatter in directions more tangent to the surface. This distribution is notably no longer
```
Comment (trevordblack) [inline books/RayTracingInOneWeekend.html:2458]: "This means that a ray is more likely to scatter in direction close to the normal, and less likely to scatter in directions close to the surface"

Alternatively:
""This means that a ray is more likely to scatter in direction close to the normal, and less likely to scatter in directions more tangent to the normal"

?

There's a couple of undesirable/awkward English A-B contrasts
"most" -> "less"
"close to the normal" -> "tangent to the surface"

Neither of my alts are perfect, not sure what is best here

Comment (hollasch) [reply books/RayTracingInOneWeekend.html]: Tweaked some more.

File: books/RayTracingInOneWeekend.html (hunk 2 above)
Comment (hollasch) [reply books/RayTracingInOneWeekend.html]: Fixed.

**Pull request merged**